typescript/infra/scripts/warp-routes/monitor-warp-routes-balances.ts

```diff
@@ -401,37 +448,35 @@ const getXERC20Limit = async (
   };
 };
 
-async function checkXERC20Limits(
+async function checkWarpRouteMetrics(
   checkFrequency: number,
   tokenConfig: WarpRouteConfig,
   chainMetadata: ChainMap<ChainMetadata>,
 ) {
   setInterval(async () => {
     try {
-      const xERC20Limits = await getXERC20Limits(tokenConfig, chainMetadata);
-      logger.info('xERC20 Limits:', xERC20Limits);
-      updateXERC20LimitsMetrics(xERC20Limits);
+      const multiProtocolProvider = new MultiProtocolProvider(chainMetadata);
+      const balances = await checkBalance(tokenConfig, multiProtocolProvider);
+      logger.info('Token Balances:', balances);
+      updateTokenBalanceMetrics(tokenConfig, balances);
     } catch (e) {
       logger.error('Error checking balances', e);
     }
-  }, checkFrequency);
-}
 
-async function checkTokenBalances(
-  checkFrequency: number,
-  tokenConfig: WarpRouteConfig,
-  chainMetadata: ChainMap<ChainMetadata>,
-) {
-  logger.info('Starting Warp Route balance monitor');
-  const multiProtocolProvider = new MultiProtocolProvider(chainMetadata);
-
-  setInterval(async () => {
     try {
-      logger.debug('Checking balances');
-      const balances = await checkBalance(tokenConfig, multiProtocolProvider);
-      updateTokenBalanceMetrics(tokenConfig, balances);
+      if (
+        Object.values(tokenConfig).every(
```

Comment:
I think for the fastUSD warp route, only the Sei side is xERC20 but the Ethereum side is a collateral warp route. I think we'd still wanna monitor the limits on Sei in this case

Reply:
makes sense